tunnistamo/settings.py

```diff
@@ -428,7 +434,18 @@
 
     # Save last login backend to user data
     'users.pipeline.save_social_auth_backend'
-)
+]
+
+# We want to have to a positive config option and NOT allowing
+# duplicate emails requires changes. Thus the inverted test.
+if not env("ALLOW_DUPLICATE_EMAILS"):
```

Comment:
I'd much rather move this logic out of the settings file. Make a new custom pipeline function which calls the original function unless env("ALLOW_DUPLICATE_EMAILS")

Reply:
Or rather, just move the condition to the existing pipeline function because it's our code anyway. I remembered it being part of library code.